### DISCUSSION TITLE
Help to Setup

### DISCUSSION THREAD

**Discussion by Hiuoy** [QIN2DIM/hcaptcha-challenger#898] [Q&A]
Anyone can guied me to setup this?
I want to implement this in my automation code, what Iam suppos to do with .onnx files like how to run them?

I have no Idea what Iam looking at, if someone can help it would be great!

**Comment by QIN2DIM** [comment]
Please describe your problem specifically

**Reply by Hiuoy** [reply]
I have this script built with puppteer, but there is the captcha blockes me from signin.
how can I build my own object detection script? what Iam suppose to do with .onnx files? how I can run them?

I searched on the internet and I found out that those are pre-trained modles.
